### PR TITLE
Switch from brentq to minimize_scalar

### PR DESCRIPTION
**Pull request by michael-hartmann**
SciPy 0.11 added a general function to compute the minimum of a scalar function, minimize_scalar. Change the disscussion from brentq to minimize_scalar.

### CODE REVIEW

File: advanced/mathematical_optimization/index.rst [hunk 1]
```diff
@@ -191,26 +207,11 @@ It combines a bracketing strategy with a parabolic approximation.
...
-    >>> x_min #doctest: +ELLIPSIS
-    0.699999999...
-    >>> x_min - .7 #doctest: +ELLIPSIS
-    -2.1605...e-10
 
 .. note:: 
    
-   Brent's method can also be used for optimization *constrained to an
-   interval* using :func:`scipy.optimize.fminbound`
+   You can use different solvers using the parameter method.
```
Comment (GaelVaroquaux) [inline advanced/mathematical_optimization/index.rst]: Maybe we want to keep and adapt the comment on "bounds": the minimize_scalar function has a "bounds" argument, doesn't it?

**Comment by GaelVaroquaux** [comment]
Doctests are failing. It looks like a minor indentation problem:

https://travis-ci.org/scipy-lectures/scipy-lecture-notes/jobs/270749938

File: advanced/mathematical_optimization/index.rst (hunk 1 above)
Comment (michael-hartmann) [reply advanced/mathematical_optimization/index.rst]: I have re-added the note on bounds.

**Comment by GaelVaroquaux** [comment]
Merging. Thank you!

**Pull request merged**